resources/js/src/app/store/modules/WishListModule.js

```diff
@@ -36,33 +38,48 @@ const mutations =
         addWishListId(state, id)
         {
             state.wishListIds.push(id);
+        },
+
+        setIsWishListInitiallyLoading(state)
+        {
+            state.isWishListInitiallyLoading = true;
+        },
+
+        setIsWishListLoading(state, isLoading)
+        {
+            state.isLoading = !!isLoading;
         }
     };
 
 const actions =
     {
-        initWishListItems({ commit }, ids)
+        initWishListItems({ commit, state })
         {
             return new Promise((resolve, reject) =>
             {
-                if (ids && ids[0])
+                if (!state.isWishListInitiallyLoading)
                 {
-                    commit("setWishListIds", ids);
+                    commit("setIsWishListInitiallyLoading");
```

Comment:
Muss das nicht eher in den success-callback? Sprich erst wenn der Call erfolgreich beendet ist?

Reply:
Wenn es da erst aufgerufen wird, würde der REST-Call 2x ausgeführt werden, wenn man die Wunschliste 2x auf der Seite hat. Daher habe ich den Wert auch "loading" und nicht "Loaded" genannt.

Reply:
Ok. Willst du dann nicht im Fehlerfall das Flag wieder zurücksetzen?